pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +157,11 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	if server.StreamingConnectionIdleTimeout == 0 {
+		cfg.StreamingConnectionIdleTimeout = 5 * time.Minute
```

Comment:
You should be able to drop this whole block. Verify cfg gets the value set by us as the default,  or the value set by them if overridden in the args